commands/tags.js

```diff
@@ -8,7 +8,12 @@ function runTag(msg) {
   if (cmd && tags.get(msg.guild.id).has(cmd.command)) {
     let tag = tags.get(msg.guild.id).get(cmd.command);
     let response = tag.response
-      .replace(/<@author>/ig, msg.author)
+    let match, regex = /\<([^)]+)\>/
```

Comment:
Why the `^)` in the regex?

Reply:
I'm not really sure. It looks like \<(.*)\> would work better.